games-server/minecraft-server/minecraft-server-1.15.2-r1.ebuild

```diff
@@ -19,26 +19,27 @@ RDEPEND="
 	acct-group/minecraft
 	acct-user/minecraft
 	app-misc/dtach
+	sys-apps/openrc
```

Comment:
What's the purpose of adding ```sys-apps/openrc``` into ```RDEPEND```? Since it's Gentoos default RC system, you don't have to add it.